app/classifier/components/Intervention/Intervention.spec.js

```diff
@@ -17,20 +17,31 @@ describe('Intervention', function () {
   };
   const onUnmount = sinon.stub()
 
-  before(function () {
-    wrapper = mount(
-      <Intervention
-        intervention={intervention}
-        onUnmount={onUnmount}
-        user={user}
-      />);
-  });
+  let wrapper = mount(
+    <Intervention
+      intervention={intervention}
+      onUnmount={onUnmount}
+      user={user}
+    />);
+
   it('should render', function () {
     expect(wrapper).to.be.ok;
   });
   it('should show a notification message', () => {
     const newlineMsg = intervention.message + "\n"
-    expect(wrapper.find(Markdown).text()).to.equal(newlineMsg);
+    expect(wrapper.find(Markdown).first().text()).to.equal(newlineMsg);
+  });
+  it('should show an opt-out message', () => {
+    const optOutHTML = '<div class="markdown "><p>I do not want to take part in this messaging ' +
```

Comment:
I'm not super happy testing the HTML here, but i wasn't sure what to do, i could check for a link `<a href=` tag vs the component html but lose some granularity over the test. 

Also for some reason there is a new line before the last div 🤷‍♂

Reply:
Yeah, this is a really brittle test.

Reply:
Could try [`contains`](https://airbnb.io/enzyme/docs/api/ShallowWrapper/contains.html) instead? I sometimes find that a bit more readable than the html render method for shallow.

Reply:
Examining the props of the Markdown component might allow you to test the markdown string that’s passed into it as `props.children`.

Reply:
Thanks that is a much nicer solution :)